main.go

```diff
@@ -22,18 +22,18 @@ func main() {
 	winnerChan := make(chan Horse)
 	for line := range board {
 		// each horse will be moved in different processes
-		go startRuningHorseInLine(board, line, winnerChan)
+		go startRunningHorseInLine(board, line, winnerChan)
 	}
 
 	winner := <-winnerChan // wait until one horse reaches the end
 	// render one last time to ensure the latest board state
 	RenderRaceBoard(board, &winner)
 
 	fmt.Println("Race finished!")
-	fmt.Printf("# Winner: %s\n", winner)
+	fmt.Printf("# Winner: %s\n", winner.Name)
```

Comment:
As last line, I would like to have the horse name and line, so I thought this would be a good opportunity for the `String()`. Using `fmt.Printf("# Winner: %s\n", winner)` was intentional, as I wanted to call to magical "toString" method which is `String()`.
```go
func (h Horse) String() string {
	return fmt.Sprintf("%s (line:%d)", h.Name, h.Line)
}
```

Reply:
Yes, then if you want to use String() then your version is correct.